batch/Dockerfile.test

```diff
@@ -8,6 +8,7 @@ RUN pip install kubernetes
 RUN pip install cerberus
 
 COPY batch /batch
+RUN pip install /batch
```

Comment:
I didn't know you could install a local python module.

Reply:
Neither did I!

Reply:
neat use!